src/main/java/cn/nukkit/inventory/DoubleChestInventory.java

```diff
@@ -102,8 +102,6 @@ public void setContents(Map<Integer, Item> items) {
                 } else if (this.right.slots.containsKey(i - this.left.size)) {
                     this.clear(i);
                 }
-            } else if (!this.setItem(i, items.get(i))) {
-                this.clear(i);
```

Comment:
I'm afraid this change needs to be reverted. I don't get why this is being removed and what this has to do with "Add most of the new nether".

Reply:
That must have been a temporary fix for a bug I encountered and was somehow put into that branch, but it is completely unrelated. I'll see if I can remove it soon